utils/unittest/execution_state.go

```diff
@@ -24,7 +24,7 @@ const ServiceAccountPrivateKeySignAlgo = crypto.ECDSAP256
 const ServiceAccountPrivateKeyHashAlgo = hash.SHA2_256
 
 // Pre-calculated state commitment with root account with the above private key
-const GenesisStateCommitmentHex = "d3518010ead6ce14c3994005fd8121691efece28232c8853ad60eaeccaed4fd2"
+const GenesisStateCommitmentHex = "4981dcc4690b2efe272ef9e5283311bf2e71472919c01fb71405fe86f991c2c2"
```

Comment:
did we check what is changing genesis state commitment, is it the new content for fee contract?

Reply:
Yes, this is because of the new contract. Not sure how I could verify that it is **only** because of the new contract.